inventory/byo/hosts.aep.example

```diff
@@ -157,6 +157,16 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # Disable the OpenShift SDN plugin
 # openshift_use_openshift_sdn=False
 
+# Configure SDN cluster network CIDR block. This network block should
+# be a private block and should not conflict with existing network
+# blocks in your infrastructure that pods may require access to.
+# Can not be changed after deployment.
+#osm_cluster_network_cidr=10.1.0.0/16
+
+# Configure master API and console ports.
+#openshift_master_api_port=8443
+#openshift_master_console_port=8443
```

Comment:
Not directly related to this, but should we eliminate the separate variables for api_port and console_port, either that or ensure that if api_port is set and console_port is not set that console_port defaults to api_port?

cc @smarterclayton